cayleypy/graphs_lib.py

```diff
@@ -81,6 +83,16 @@ def prepare_graph(name, n=0) -> CayleyGraph:
             generators.append(perm)
             generator_names.append("R" + str(prefix_len - 1))
         return CayleyGraph(generators, dest=list(range(n)), generator_names=generator_names)
+    elif name == "burnt_pancake":
+        assert n >= 1
+        generators = []
+        generator_names = []
+        for prefix_len in range(0, n):
+            perm = list(range(n+prefix_len,n-1,-1)) + list(range(prefix_len+1,n,1)) + \
```

Comment:
Do not use a backslash for explicit line continuation: https://google.github.io/styleguide/pyguide.html#32-line-length

There is a nicer way to write this:
```
perm = []
perm += list(range(n+prefix_len,n-1,-1)) 
perm += list(range(prefix_len+1,n,1)) 
perm += list(range(prefix_len,-1,-1))
perm += list(range(n+prefix_len+1,2*n,1))
```